lang/src/org/partiql/lang/eval/physical/operators/AggregateOperatorFactory.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ * A copy of the License is located at:
+ *
+ *      http://aws.amazon.com/apache2.0/
+ *
+ *  or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+ *  language governing permissions and limitations under the License.
+ */
+package org.partiql.lang.eval.physical.operators
+
+import org.partiql.lang.domains.PartiqlPhysical
+
+public abstract class AggregateOperatorFactory(name: String) : RelationalOperatorFactory {
+    public final override val key: RelationalOperatorFactoryKey = RelationalOperatorFactoryKey(RelationalOperatorKind.AGGREGATE, name)
+    public abstract fun create(impl: PartiqlPhysical.Impl): AggregateOperator
```

Comment:
Did something essentially very similar, but the AggregateOperator is now removed, since it's the same thing as a RelationExpression. Looks similar to the rest of the implementations.